README.md

```diff
@@ -0,0 +1,41 @@
+# Cincinnati
+
+Cincinnati is a service to transform update information as modeled in a Docker registry (such as Quay.io) into a graph representation that can be consumed by update clients. It is the spiritual successor to the Omaha protocol used to udate Google Chrome and Google Chrome OS based operating systems.
+
+## Common Usage
+
+```sh
+# docker run -d -p 8080:8080 <image> --address 0.0.0.0 --registry https://quay.io --repository <namespace/reponame>
+# docker run -d -p 8081:8081 --entrypoint policy-engine <image> --address 0.0.0.0 --upstream http://localhost:8080/v1/graph
+# curl -v -H 'Accept:application/json' http://localhost:8080/graph
+```
+
+## Components
+
+### Graph Bulder
+
+The Graph Builder is responsible for transforming the data in a Docker registry repository into a directed acyclic graph (DAG) describing update payloads as nodes, and their relationship as edges. 
```

Comment:
"...(DAG), describing update payloads as nodes and valid update transitions as edges."